src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
-        }
-      />
-      <Page.Body
-        loading={loading || cloning}
-        retryOnError={() => {
-          return
-        }}
-        error={(error?.data as Error)?.message}
-      >
-        <GetStarted isOpen={isOpen} setDrawerOpen={val => setDrawerOpen(val)} />
-        <Layout.Vertical padding="large" background={Color.GREY_0}>
+    <Page.Body
+      loading={loading || cloning}
+      retryOnError={() => {
+        return
+      }}
+      error={(error?.data as Error)?.message}
+    >
+      <Layout.Vertical padding="large" background={Color.GREY_0}>
+        <Layout.Horizontal
+          style={{
+            borderBottom: '1px solid var(--grey-100)',
+            justifyContent: 'space-between',
+            paddingLeft: 0,
+            paddingRight: 0
+          }}
+          flex={true}
+        >
           <Layout.Horizontal
-            style={{
-              borderBottom: '1px solid var(--grey-100)',
-              justifyContent: 'space-between',
-              paddingLeft: 0,
-              paddingRight: 0
-            }}
-            flex={true}
+            spacing="medium"
+            style={{ justifyContent: 'space-between', alignItems: 'center', width: '100%' }}
           >
-            <Layout.Horizontal
-              spacing="medium"
-              style={{ justifyContent: 'space-between', alignItems: 'center', width: '100%' }}
-            >
-              <section style={{ display: 'flex' }}>
-                <RbacButton
-                  intent="primary"
-                  text={getString('dashboardLabel')}
-                  onClick={() => showModal()}
-                  icon="plus"
-                  className={css.createButton}
-                  permission={permissionObj}
-                />
+            <section style={{ display: 'flex' }}>
+              <RbacButton
+                intent="primary"
+                text={getString('dashboardLabel')}
+                onClick={() => showModal()}
+                icon="plus"
+                className={css.createButton}
+                permission={permissionObj}
+              />
 
-                <Layout.Horizontal className={css.predefinedTags + ' ' + css.mainNavTag}>
-                  <ModuleTagsFilter selectedFilter={selectedFilter} setPredefinedFilter={setPredefinedFilter} />
-                </Layout.Horizontal>
-              </section>
+              <Layout.Horizontal className={css.predefinedTags + ' ' + css.mainNavTag}>
+                <ModuleTagsFilter selectedFilter={selectedFilter} setPredefinedFilter={setPredefinedFilter} />
+              </Layout.Horizontal>
+            </section>
+            <Layout.Horizontal>
+              <CustomSelect
+                items={sortingOptions}
+                filterable={false}
+                itemRenderer={(item, { handleClick }) => (
+                  <div key={item.value.toString()}>
+                    <Menu.Item text={item.label} onClick={handleClick} />
+                  </div>
```

Comment:
Is this surrounding `div` required?
```suggestion
                  <Menu.Item
                    key={item.value.toString()}
                    text={item.label}
                    onClick={handleClick}
                  />
```